.github/workflows/publish-operator.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish Dev Operator
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
```

Comment:
can you push a tag like `v1.0.0-test` to test if this actually works?

Reply:
added an entry:

`      - 'v*.*.*-*'`

Reply:
worked now: https://github.com/ninech/tailscale/pkgs/container/tailscale